tests/conftest.py

```diff
@@ -4527,6 +4527,34 @@ def data_context_with_datasource_spark_engine(empty_data_context, spark_session)
     return context
 
 
+@pytest.fixture
+def data_context_with_datasource_spark_engine_batch_spec_passthrough(
+    empty_data_context, spark_session
+):
+    context = empty_data_context
+    config = yaml.load(
+        f"""
+    class_name: Datasource
+    execution_engine:
+        class_name: SparkDFExecutionEngine
+    data_connectors:
+        default_runtime_data_connector_name:
+            class_name: RuntimeDataConnector
+            batch_identifiers:
+                - default_identifier_name
+            batch_spec_passthrough:
+                reader_method: csv
+                reader_options:
+                    header: True
+        """,
+    )
+    context.add_datasource(
+        "my_datasource",
+        **config,
+    )
+    return context
+
```

Comment:
This is used for testing whether the the `batch_spec_passthrough` parameters will work if added to the `Datasource` configuration, rather than `BatchRequest`